src/components/NotFoundPage/NotFoundPageLayout.js

```diff
@@ -12,3 +12,5 @@ export const NotFoundPageLayout = styled('div')`
   right: 0;
   margin: auto;
 `;
+
+export default NotFoundPageLayout;
```

Comment:
looks like i forgot to delete this file. you can go ahead and remove it and remove the dependencies as well